### PR TITLE
Integrate User Avatar in Settings #1837, Create getUserAvatarLink utility #2154

### LINKED ISSUE #1837: Integrate User Avatar in User Account Section with Redux
**Actual Behaviour**

Not present, user should be able to add an avatar, and the avatar should be setup in redux state, so that TopBar and Static Bar bar can use it

**Expected Behaviour**

Integrate User Avatar in User Account Section with Redux

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1837, #2154 

Changes:
 - Integrate User Avatar in User Account Section with Redux
 - Created getUserAvatarLink utility function #2154

Demo Link: https://pr-2157-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -8,3 +8,5 @@ export { default as scrollAnimation } from './scrollAnimation.js';
 export { default as testExample } from './testExample';
 export { default as parseDate } from './parseDate';
 export { default as formatDate } from './formatDate';
+export { default as getGravatarProps } from './getGravatarProps';
```
Comment (akshatnitd) [inline src/utils/index.js]: Let's keep avatar related utils in 1 file.

File: src/utils/getUserAvatar.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { urls } from './';
+
+let getUserAvatar = accessToken => {
```
Comment (akshatnitd) [inline src/utils/getUserAvatar.js]: Also, rename it to getUserAvatarLink

**Comment by fragm3** [comment]
@akshatnitd I have added location.reload to reload the avatar, please test it once, working fine. Couldn't think of another cleaner way to get this done as avatar image depends on timestamp.

**Comment by fragm3** [comment]
@akshatnitd I will be taking one more issue to fix the staticAppBar, topAppBar with the store properly, created an issue for it https://github.com/fossasia/susi.ai/issues/2160.

**Comment by fragm3** [comment]
@akshatnitd made the changes, works now

**Pull request merged**